### DISCUSSION TITLE
123云盘挂载失败

### DISCUSSION THREAD

**Discussion by wjn6** [AlistGo/alist#2673] [Q&A]
显示failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field Files.data of type struct { InfoList []_123.File "json:\"InfoList\""; Next string "json:\"Next\"" }

**Comment by Shunamxiao** [comment]
failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field Files.data of type struct { InfoList []_123.File "json:\"InfoList\""; Next string "json:\"Next\"" }
